src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/MissingDeprecatedCheck.java

```diff
@@ -113,6 +116,9 @@ public final class MissingDeprecatedCheck extends AbstractJavadocCheck {
     public static final String MSG_KEY_ANNOTATION_MISSING_DEPRECATED =
             "annotation.missing.deprecated";
 
+    /** Message property key for the Unclosed HTML message. */
+    public static final String MSG_KEY_UNCLOSED_HTML_TAG = "javadoc.unclosedHtml";
```

Comment:
I am confused by this.

This issue is just about fixing the violation message for users. #13775 is for fixing the underlying issue. This field isn't even used in this class as shown by changes. When #13775 is fixed as described, this field will need to be removed. So I don't see a purpose to add this field.

We should just keep property changes, and example changes if possible and revert the rest.
@nrmancuso @romani

Reply:
@rnveach When i remove `MSG_KEY_UNCLOSED_HTML_TAG` yet keep everything else as it is, one of the test case fails as mentioned below:
```
[ERROR] Failures: 
[ERROR]   AllChecksTest.testAllCheckstyleMessages:572 property 'javadoc.unclosedHtml' isn't used by any check in package 'com.puppycrawl.tools.checkstyle.checks.annotation'
expected to contain: javadoc.unclosedHtml
but was            : [annotation.incorrect.style, annotation.parens.missing, annotation.trailing.comma.present, annotation.parens.present, annotation.trailing.comma.missing, suppressed.warning.not.allowed, annotation.location.alone, annotation.location, javadoc.parse.rule.error, javadoc.duplicateTag, annotation.missing.deprecated, javadoc.wrong.singleton.html.tag, javadoc.missed.html.close, tag.not.valid.on, annotation.missing.override, annotation.package.location, annotation.same.line]
```

And, @nrmancuso mentioned above that this test doesn't have any suppression. Please let me know your feedback on the same.

Reply:
There is a basic suppression method at 
https://github.com/checkstyle/checkstyle/blob/351bbc1ecf4cdcf158b5295c2b2f1c602cdbfc8a/src/test/java/com/puppycrawl/tools/checkstyle/internal/AllChecksTest.java#L564-L567

Reply:
I am ok to extend this condition until #13775

Reply:
@rnveach , if we apply your change, it becomes a problem for macros to generate proper xdoc, as this message becomes not visible.

Reply:
That's expected as this issue is not about fixing the xdoc. The other one is. This issue is described as  to just fix the violation message.

Reply:
If code changes already done, there is no hacks, code will be same as after next issue, why we need to remove it? And extended test with extra suppression.

Reply:
here is version you think is good  - https://github.com/checkstyle/checkstyle/pull/13808

please merge what is good for you.